yarn.lock

```diff
@@ -13232,10 +13288,10 @@ statuses@~1.4.0:
   resolved "https://registry.yarnpkg.com/statuses/-/statuses-1.4.0.tgz#bb73d446da2796106efcc1b601a253d6c46bd087"
   integrity sha512-zhSCtt8v2NDrRlPQpCNtw/heZLtfUDqxBM1udqikb/Hbk52LK4nQSwr10u77iopCW5LsyHpuXS0GnEc48mLeew==
 
-stellar-base@8.2.2-soroban.12:
-  version "8.2.2-soroban.12"
-  resolved "https://registry.yarnpkg.com/stellar-base/-/stellar-base-8.2.2-soroban.12.tgz#81392ae6ea5c8b2316faacd2304a13c23ec850db"
-  integrity sha512-bQQxRFA2GXoNsr2ZFN+eK9Du1hSu0+t983tHRMI3nyWlLgZInKg7B9ZFa4UjLbLYpxQ4mE1EYupk3bulqNA91A==
+stellar-base@^8.0.0:
+  version "8.0.0"
+  resolved "https://registry.yarnpkg.com/stellar-base/-/stellar-base-8.0.0.tgz#619e5fb7951fa8abb26322e51e611169a5d1cf62"
+  integrity sha512-MH0V94Hn/Ekq/yZxd+EQijkvfU+21zuqJAyEP8X+fhRR0IQYwGNByY003oLqIkgwqXotyofEQ76W5MoNFZmlpw==
```

Comment:
uhhh???

Reply:
Yeah, this *may* be just some weirdness with how yarn.lock is generated/how GH diffed. If you look below, stellar-base@^8.0.0 used to be down there, but it just pulled it up higher in yarn.lock, and put stellar-base@^9.0.0-soroban.1 down where stellar-base@^8.0.0 used to be